### DISCUSSION TITLE
IU봇의 이름을 변경할 것을 제안드립니다.

### DISCUSSION THREAD

**Discussion by Entick-OP** [koreanbots/koreanbots#233] [제안] [answered]
디스코드 서버에 있는 공식 봇 'IU'의 이름의 변경이 필요하다 생각됩니다.
장기간 디스코드 서버에 있는 사용자들은 사이트 개발자 원더님이 IU의 팬이신걸 알고, IU 봇을 통해 문의를 하면 된다는걸 알지만 신규 이용자들에게는 IU의 팬 봇처럼 보일 가능성이 높아보입니다.
'한국 디스코드봇 리스트 공식 봇'  등 공식 봇임을 잘 알릴 수 있는 직관적인 이름으로 변경하면 좋겠습니다

**Comment by wonderlandpark** [comment, marked as answer]
안녕하세요, 제안해주신 내용에 대해 한국 디스코드봇 리스트 관리진간에 상의해보았습니다.

최근 이에 대해 의견도 있었기에 이곳을 통해 입장을 밝힙니다

1. 해당 봇은 한국 디스코드봇 리스트 서버에서만 사용되는 봇이므로 봇 이름이 크게 상관은 없을듯하다.

1. 유사한 봇 리스트 디스코드의 경우 다른 형태의 네이밍을 사용하긴 하지만 DBL은 Luca를 Discord Bots는 cake를 사용한다.

1. 봇의 작명은 개발자의 역량이며, 봇이 최상단에 위치하고 있어 충분히 공식봇임을 알 수 있다.

또한, 현재는 디스코드봇으로 문의를 받고 있지만 이후에 웹으로 문의 시스템을 이전할 계획도 갖고 있습니다.

또 어디로 문의할지 모를 수도 있다는 의견도 주셨는데, 현재 많은 유저분들이 IU메일로 문의를 해주시고 계십니다. 관리자 디엠 문의도 별로 오지 않고 `#문의` 채널을 닫은 현재로써는 별 필요성이 안 느껴집니다.

하나 더 덧붙이자면 아이유봇은 가장 초창기의 한국 디스코드봇 리스트의 시작입니다. 따라서 전 이 역사를 유지하고 싶습니다.

또한, 봇 이름조차 제가 결정권을 갖지 못한다면 한국 디스코드봇 리스트 개발의 소소한 재미를 잃을거 같습니다.

감사합니다
